README.md

```diff
@@ -75,6 +81,13 @@ Hardhat Node, Foundry Anvil or sapphire-dev docker image, don't forget to
 *clear your account's activity* each time or manually specify the correct
 account nonce.
 
+When click button Connect Wallet, in some case you will constantly get the error: 
+**Uncaught (in promise) Error: [useEthereumStore] Request account failed!**, 
+
+To resolve it, try to open MetaMask and manually connect site once time
+
+![alt text](manually-connect-site-1.png)  ![alt text](manually-connect-site-2.png)
+
```

Comment:
I cannot reproduce this one and it seems as a MetaMask-specific error. I would either not put this into README.md at all or add a new `## Troubleshooting` at the end of the README.md with the description above.

Please place the two screenshots into the new docs/ folder.

Reply:
I added a new ## Troubleshooting at the end of the README.md with the description above & moved two screenshots into the new docs/ folder